packages/gatsby-theme-project-portal/src/layouts/SearchPageLayout.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import { SearchPageLayout } from "./SearchPageLayout"
+import * as ProjectTeamStories from "../components/ProjectTeam.stories"
+
+const meta: Meta<typeof SearchPageLayout> = {
+  component: SearchPageLayout,
+  tags: ["autodocs"],
+}
+
+export default meta
+
+type Story = StoryObj<typeof SearchPageLayout>
+
+export const Primary: Story = {
+  args: {
+    data: {
+      allGeneralPage: {
+        nodes: {
+          slug: "/about",
```

Comment:
I think you might be showing the slug rather than the page title at the top of the results card for these general pages:
![Screenshot 2023-10-26 at 16 34 51](https://github.com/thepolicylab-projectportals/project-portal-theme/assets/2803227/e15379fd-dfc1-4e9f-a6b9-f3bb6b4136fe)